agents.ipynb

```diff
@@ -16,7 +16,7 @@
   },
   {
```

Comment:
Please ignore the changes in agents.ipynb

Reply:
I think you need to undo those commits yourself. `git cherry-pick` isn't always reliable